backend/config/routes.rb

```diff
@@ -208,6 +208,8 @@
         end
       end
     end
+
+    resources :style_guide
```

Comment:
Since you only need the `index` route I would either use `only: [:index]` or just use `get` here. The additional routes rails generates are not harmful, but clutter up `rake routes` and could lead to confusion.